### ISSUE TITLE
Tool is not deployed 

### ISSUE THREAD

**Issue by MikDal002** [advinst/teamcity-advinst-build-runner#6]
Hello,

I'm adding Advanced Installer as a build step in TeamCity. On first agent, everything works like a charm. But, unfortunately, the second machine won't work. I came across the problem that  the advanced installer command line tool is not deployed. 

Error is: 

```
D:\TeamCity\buildAgent\work\b8ed9bda369407fa\Installer\Installer.aiproj(35,5): error : This project requires Advanced Installer tool. Please download it from https://www.advancedinstaller.com/download.html
```

and this happens during general project build with command: 
```
"C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\MSBuild.exe" D:\TeamCity\buildAgent\work\b8ed9bda369407fa\Solution.sln /p:Configuration=Debug @D:\TeamCity\buildAgent\temp\agentTmp\1.rsp -property:Platform=x64
```

My build steps are in that order: 
![image](https://user-images.githubusercontent.com/25830778/169479270-2d878100-dddb-4b08-99a8-07482ab3bb2e.png)

The first step was added when the error occurred and is just about deploying:

![image](https://user-images.githubusercontent.com/25830778/169479429-9447a019-19ef-408e-ba8a-7d59b0226559.png)
I have tried to build with "Enable PowerShell support" enabled and without.

And here is how the installer is configured. 
![image](https://user-images.githubusercontent.com/25830778/169479666-f817dc39-dd94-4fcb-a296-5ac25e3eb10a.png)

Normally, on my first build agent everything works fine, even without first step (that about deploying). But on the second agent, building installer completely doesn't work. Do I have to deploy the tool manually? 

I can confirm, that the Advanced Installer is deployed to machine in «build agent directory»/tools/advancedinstaller-19.4 

![image](https://user-images.githubusercontent.com/25830778/169484874-3d9d1e60-0996-4d1e-899d-3b80ceb38465.png)

So I completely do not understand why I cannot build the installer. Could you help me? :)

**Comment by burcadoruciprian** [comment]
Hey, 
Currently the TeamCity extension cannot build .aiproj projects. As you noticed, the tool is indeed deployed on the agent but the .aiproj cannot discover it. My guess is that that, on the first agent you have a pre-existing instance of the advinst tool, which you installed manually.

We will add support for building .aiproj files in the next version.

**Comment by MikDal002** [comment]
It is good to hear, that functionality will appear in next version. When can it be expected? 🙂

**Comment by burcadoruciprian** [comment]
We will strive to make it sooner rather than later :). I cannot give you an eta though

**Comment by burcadoruciprian** [comment]
Hello,
I have updated the plugin to build .aiproj files. Check it out and let me know if you encounter any issues.

**Comment by MikDal002** [comment]
It is good to hear, that functionality will appear in next version. When can it be expected? 🙂
________________________________
Od: Ciprian Burca ***@***.***>
Wysłane: poniedziałek, 23 maja 2022 08:31
Do: Caphyon/teamcity-advinst-build-runner ***@***.***>
DW: MikDal002 ***@***.***>; Author ***@***.***>
Temat: Re: [Caphyon/teamcity-advinst-build-runner] Tool is not deployed (Issue #6)


Hey,
Currently the TeamCity extension cannot build .aiproj projects. As you noticed, the tool is indeed deployed on the agent but the .aiproj cannot discover it. My guess is that that, on the first agent you have a pre-existing instance of the advinst tool, which you installed manually.

We will add support for building .aiproj files in the next version.

**Comment by burcadoruciprian** [comment]
This functionality is available since 1.4 version (june 6)
